doc/guides/zephyr_cmake_package.rst

```diff
@@ -382,6 +382,48 @@ Such a CMakeLists.txt could look as:
 
    project(my_first_app)
 
+.. _cmake_extension_package:
+
+Zephyr Extension CMake package
+******************************
+
+The Zephyr Extension CMake package provides a possibility for a Zephyr based system to control
+Zephyr build settings in a generic way.
+The Zephyr Extension CMake package will be loaded in the Zephyr boilerplate code after initial
+properties and ``ZEPHYR_BASE`` has been defined, but before CMake code execution.
+This allows the Zephyr Extension CMake package to setup or extend properties such as: ``DTS_ROOT``,
+``BOARD_ROOT``, Fixed overlays, and any other property that can be controlled.
+
+To provide a Zephyr Extension CMake package, create ``ZephyrExtensionConfig.cmake`` and place it in
+a Zephyr workspace top-level folder as:
+
+.. code-block:: none
+
+   <projects>/zephyr-workspace
+   ├── zephyr
+   ├── ...
+   └── zephyr_extension_project (can be named anything)
```

Comment:
I would use a name that is less confusing, i.e. 'zephyr application', from first glance it reads like I need to create a project for the extension package...

Reply:
good point.
Main point is that the package can be in any project.

Will rephrase.

Reply:
Done, also read the updated docs.